docs/spec/manifest-v2-2.md

```diff
@@ -89,6 +89,11 @@ image manifest based on the Content-Type returned in the HTTP response.
             The os field specifies the operating system, for example
             `linux` or `windows`.
 
+        - **`osversion`** *string*
```

Comment:
I'm open to that, but it's a little strange since `.` is the field separator in most languages. This means that if you were manipulating this in JavaScript, for example, you couldn't write `p.os.version`, you'd have to write `p["os.version"]`. But AFAIK, if it's osversion, you can just write `p.osversion`.